### DISCUSSION TITLE
UI Bug on bottom Menu

### DISCUSSION THREAD

**Discussion by zcuric** [community/community#36224] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info

When accidentally swipping left, but  not completing the swipe, this happened. 

![image](https://user-images.githubusercontent.com/1560102/195976749-897b775e-c4e1-427d-abe1-b1f499fc4676.png)
</summary>
GitHub for iOS, Version 1.84.0 (80448590)
Device: iPhone 13 Pro (iOS 16.0.2)
</details>

**Comment by iskae** [comment]
Hey @zcuric 👋 

Thanks for reaching out and filing a bug report ❤️ 

We are able to reproduce it and will be focusing on fixing it in the future releases 🎉 Please stay tuned 😊 

Cheers!

**Reply by Kuuhaku1010** [reply]
How can i contact u mr?
